Library/Formula/jigdo.rb

```diff
@@ -18,7 +18,7 @@ class Jigdo < Formula
 
   # Use MacPorts patch for compilation on 10.9; this software is no longer developed.
   patch :p0 do
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!